doc/index.md

```diff
@@ -0,0 +1,23 @@
+Welcome to the OCurrent wiki!
+
+### Introduction
+
+* [Example pipelines](example_pipelines.md)
+* [Skeleton project](https://github.com/ocurrent/ocurrent-skeleton) - a template for new projects
+* [Internals](internals.md) - how it all works
+* [API docs](https://ocurrent.github.io/ocurrent/index.html)
+
+### OCurrent plugins
+
+* [Current_docker](https://ocurrent.github.io/ocurrent/current_docker/index.html)
+* [Current_fs](https://ocurrent.github.io/ocurrent/current_fs/index.html)
```

Comment:
Broken link.

Reply:
`Current_fs` doesn't get exposed as a library with an opam file.
Should it?

Reply:
We might split it out later: see https://github.com/ocurrent/ocurrent/pull/276#issuecomment-867484381

Reply:
I'll point to the code and this can be updated when that happens.